### DISCUSSION TITLE
"syntax error near unexpected token `newline'" error in initialising a project

### DISCUSSION THREAD

**Discussion by saeid93** [kubernetes-sigs/kubebuilder#2888] [Support] [answered]
I'm following https://book.kubebuilder.io/quick-start.html for installing the kubebuilder. After installation I tried the following for initiallizing the guestbook:
```
mkdir -p ~/projects/guestbook
cd ~/projects/guestbook
kubebuilder init --domain my.domain --repo my.domain/guestbook
```
Resulted in the following error:
```
kubebuilder init --domain my.domain --repo my.domain/guestbook
/usr/local/bin/kubebuilder: line 8: syntax error near unexpected token `newline'
/usr/local/bin/kubebuilder: line 8: `<!DOCTYPE html>'
```

What might be causing this?

**Comment by camilamacedo86** [comment, marked as answer]
We are unable to reproduce this issue. We need more info to try help you out.
 
See that have not ``<!DOCTYPE html>'` in the scaffolds and it seems something in your directory.
Ideally, you should not create the project in a dir that has other files already. That might be the issue. 

If you still facing problems, please raise an issue and describe the versions as your env for we have further info to try to help you out. Making this one as answered.

**Comment by camilamacedo86** [comment]
Closing since it is answered.

**Comment by joshwright10** [comment]
I also just had this issue. It was due to Go not being available on my path when I ran the kubebuilder installation steps. 
Check that Go is installed and available on your path before running the kubebuilder install.

You can run this to check that Go is available: 
```bash
go version
```
